packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -60,8 +60,17 @@
             {{/if}}
         
         "status":{{>ValueSet/ObservationStatus.hbs code=observationEntry.statusCode.code}},
-        "code": {{>DataType/CodeableConcept.hbs code=observationEntry.code text=observationEntry.text._}},
-        
+
+        {{#if observationEntry.code.code}}
+          "code": {{>DataType/CodeableConcept.hbs code=observationEntry.code text=observationEntry.text._}},
+        {{else}}
+          {{#if observationEntry.value.length}}
+            "code": {{>DataType/CodeableConcept.hbs code=observationEntry.value.[0] text=observationEntry.text._}},
```

Comment:
0 indexing but what if its not an array or empty?

Reply:
ditto removed this